test/externalTests/solc-js/solc-js.sh

```diff
@@ -35,7 +35,7 @@ function solcjs_test
     SOLCJS_INPUT_DIR="$TEST_DIR"/test/externalTests/solc-js
 
     # set up solc-js on the branch specified
-    setup "$SOLJSON" master
+    setup "$SOLJSON" smt_test_use_engine
```

Comment:
Just a reminder to switch this back to `master` now that https://github.com/ethereum/solc-js/pull/521 is merged.

Reply:
Yep, just removed it